From 7bbe23b71b8b967c24c446a644b0a8597fa2f384 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Wed, 25 Oct 2017 04:29:21 +0300
Subject: [PATCH 1262/1345] crypto: inside-secure: remove unused return code

commit  4214e3c5b2c23f85ec13bac438b1647183b6ebeb from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

safexcel_dequeue is void and the return code in never
used, can be removed

Change-Id: I8be1af66264838cdba502de67ce51cbc1819c38a
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/45571
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/51643
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/crypto/inside-secure/safexcel.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/crypto/inside-secure/safexcel.c b/drivers/crypto/inside-secure/safexcel.c
index 5471392..6718d82 100644
--- a/drivers/crypto/inside-secure/safexcel.c
+++ b/drivers/crypto/inside-secure/safexcel.c
@@ -701,10 +701,8 @@ void safexcel_dequeue(struct safexcel_crypto_priv *priv, int ring)
 		}
 
 		request = kzalloc(sizeof(*request), EIP197_GFP_FLAGS(*req));
-		if (!request) {
-			ret = -ENOMEM;
+		if (!request)
 			goto resource_fail;
-		}
 
 		ctx = crypto_tfm_ctx(req->tfm);
 		ret = ctx->send(req, ring, request, &commands, &results);
-- 
1.7.9.5

